data/src/main/java/me/androidbox/data/remote/model/request/EventRequestDto.kt

```diff
@@ -0,0 +1,21 @@
+package me.androidbox.data.remote.model.request
+
+
+/**
+ * /event
+ * POST/Multipart
+ *
+ * Creates a new event. Since Tasky supports offline mode, the IDs for agenda items
+ * need to be generated client-side. Please use UUID.randomUUID().toString() for that.
+ *
+ * Response body: EventDto
+ * */
+data class EventRequestDto(
+    val id: String,
+    val title: String,
+    val description: String,
+    val from: Long,
+    val to: Long,
+    val remindAt: Long,
+    val attendeeIds: List<String>
```

Comment:
I recommend adding SerializedName annotation if using GSON, or  @field:Json if using Moshi

If not, if you end up changing the variable name, or the app gets encrypted (you release it to playstore), then this Dto won't work